moveit_ros/planning/trajectory_execution_manager/test/test_execution_manager.test

```diff
@@ -1,20 +1,37 @@
-<?xml version="1.0"?>
 <launch>
-  <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
-  <include file="$(find moveit_resources)/pr2_moveit_config/launch/planning_context.launch">
-    <arg name="load_robot_description" value="true"/>
-  </include>
+    <!-- Specify the config files to use -->
+    <rosparam ns="test_execution_manager" command="load" file="$(find moveit_ros_planning)/trajectory_execution_manager/test/config/moveit_cpp.yaml" />
```

Comment:
Dropping the namespace specification in the code above, you can drop it here (and a few other locations below) as well, which simplifies code:
```suggestion
    <rosparam command="load" file="$(find moveit_ros_planning)/trajectory_execution_manager/test/config/moveit_cpp.yaml" />
```

Reply:
okay